routers/record.go

```diff
@@ -75,6 +76,18 @@ func AfterRecordMessage(ctx *context.Context) {
 		return
 	}
 
+	urlPath := getUrlPath(ctx.Request.URL.Path)
```

Comment:
urlPath has been parsed in NewRecord function, is there any bug that you must add these if judgement?